crypto/data_formats_test.go

```diff
@@ -16,14 +16,93 @@ package crypto
 
 import (
 	"bytes"
+	"reflect"
 	"testing"
 
 	"github.com/google/trillian"
 	"github.com/google/trillian/testonly"
+	"github.com/kr/pretty"
 )
 
 var dh = testonly.MustHexDecode
 
+func TestLogRoot(t *testing.T) {
+	for _, logRoot := range []*LogRootV1{
+		{
+			Version:  1,
+			RootHash: []byte("foo"),
+			Metadata: []byte{},
+		},
+	} {
+		b, err := SerializeLogRoot(logRoot)
+		if err != nil {
+			t.Errorf("SerializeLogRoot(%v): %v", logRoot, err)
+		}
+		r2, err := ParseLogRoot(b)
```

Comment:
Just call this `got` and skip the aliasing below.

Reply:
Done, but are we sure that skipping the aliasing best matches go style?

Reply:
I didn't think the aliasing was the point, more that things are named something-like "got" and "want".  E.g. the [example in the docs](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures) doesn't introduce a `if want := tt.want; ...` alias